lib/ansible/cli/doc.py

```diff
@@ -312,10 +312,9 @@ class DocCLI(CLI, RoleMixin):
     _RULER = re.compile(r"\bHORIZONTALLINE\b")
 
     # rst specific
-    _REFTAG = re.compile(r":ref:")
-    _TERM = re.compile(r":term:")
-    _NOTES = re.compile(r".. note:")
-    _SEEALSO = re.compile(r"^\s*.. seealso:.*$", re.MULTILINE)
+    _RST_ADMONITION = re.compile(r".. (\w+?)::")
```

Comment:
This doesn't match admonitions but any directive.
It could be
```rst
.. include:: some-other/doc.rst
```
or
```rst
.. code-block:: python
```
I'm not sure that you'd want to replace everything with `Note:`. I'd stick with only matching admonitions (as listed at https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions) because they are semantic directives for note blocks while other directives may have custom behaviors. Some of the directives may only generate content conditionally, contain metadata or do nothing at all. Like a spelling word list that sphinxcontrib-spelling provides.